modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -58,50 +78,50 @@ sshKey: <public_ssh_key> <5>
 pullSecret: '<pull_secret>' <6>
 # ...
 ----
-<1> The base domain of your cloud provider. 
-<2> The IP address from the VCN that the CIDR allocates to resources and components that operate on your network.
+<1> The base domain of your cloud provider.
+<2> The IP address from the  virtual cloud network (VCN) that the CIDR allocates to resources and components that operate on your network.
 <3> Depending on your infrastructure, you can select either `x86_64`, or `amd64`.
-<4> Set `OCI` as the external platform, so that {product-title} can integrate with {oci}. 
+<4> Set `OCI` as the external platform, so that {product-title} can integrate with {oci}.
 <5> Specify your SSH public key.
-<6> The pull secret that you need for authenticate purposes when downloading container images for {product-title} components and services, such as Quay.io. See link:https://console.redhat.com/openshift/install/pull-secret[Install {product-title} 4] from the Red Hat Hybrid Cloud Console. 
+<6> The pull secret that you need for authenticate purposes when downloading container images for {product-title} components and services, such as Quay.io. See link:https://console.redhat.com/openshift/install/pull-secret[Install {product-title} 4] from the Red{nbsp}Hat Hybrid Cloud Console.
 
-. Create a directory on your local system named `openshift`. 
+. Create a directory on your local system named `openshift`.
 +
 [IMPORTANT]
 ====
 Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
 ====
 
-. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, select the **<> Code** button and click **Download ZIP**. Save the archive file to your `openshift` directory,  so that all the {oci-ccm-full} and {oci-csi-full} manifests exist in the same directory. The downloaded archive file includes files for creating cluster resources and custom manifests. 
+. From the link:https://github.com/oracle-quickstart/oci-openshift[`oracle-quickstart / oci-openshift`] GitHub web page, select the *<> Code* button and click *Download ZIP*. Save the archive file to your `openshift` directory,  so that all the {oci-ccm-full} and {oci-csi-full} manifests exist in the same directory. The downloaded archive file includes files for creating cluster resources and custom manifests.
```

Comment:
GitHub has been approved at the progam-level by Ju Lim.

Reply:
@sferich888 Hi Eric, looks like we had a third-party GitHub link to Oracle approved by the program previously. As far as we know the only way to do this Oracle install is to get the files from Oracle's GitHub repo. Have you approved this, and if not could you approve it or let us know if there are concerns? Thank you.

Reply:
@julienlim  what keeps Oracle from changing these files (and breaking installs for OCP customers), incidentally? If there are issues with these files; who is responsible for fixing them (Red Hat or Oracle), and under what SLA's? 

We don't allow github links like this in the docs; because the content isn't RCMed and thus aligned and tested per release. Dynamic content like this creates all kinda of potential security issue (supply chain attacks, etc), and confusion with customers (about who/what is responsible for the content if there are issues - in supporting the installation). 

For these reasons I don't approve of links or content like this. If the BU is going to side step this guidance; they have can do so by assuming the risks involved in these situations; but should do so continuously and properly calculate the ramifications of agreeing to such risks.

Reply:
My recommendation is to not us the github links; but to have docs include the ~4 files necessary for a proper installation directly into our docs.

Reply:
@sferich888 This is part of the signed BYOS agreement. Let's chat offline.

Reply:
Thanks folks for getting that conversation going. 

This is existing content that I noticed while reviewing @dfitzmau's changes since the lines were adjacent. This PR does not add the GH repo links, so I think it is fine to go ahead merging the actual changes in this PR. Any GH link related changes can be handled in a new PR after everyone has a chance to talk this through. 

@mwerner2113 does that make sense to you as well?

Reply:
Makes perfect sense to me @jeana-redhat @dfitzmau